docs/src/main/sphinx/connector/polymorphic-table-function-ordering.fragment

```diff
@@ -0,0 +1,5 @@
+.. note::
+
+    Polymorphic table functions may not preserve the order of the query result.
+    Queries that contain an ``ORDER BY`` clause that are then encapsulated
```

Comment:
This should be rephrased as relating to the "query" function in connectors. The concept of ORDER BY (as intended is this PR), is not a generic concept that applies to every table function, but only to functions meant to execute user-provided SQL on other databases.

Also, the file should be renamed to reflect this. Maybe `query-table-function-ordering.fragment`

Reply:
problem is also that the "query" table function is NOT using that name everywhere..

Reply:
What Manfred said. The solution in this case is just that the fragment is only showing up under the query (or equivalent) function in each connector doc, not under the higher-level "table functions" section.

Reply:
>problem is also that the "query" table function is NOT using that name everywhere.

@mosabua as far as I know, the only exception is `raw_query` in Elasticsearch. This fragment should _not_ go there anyway, as Elasticsearch uses quite different syntax.

@jhlodin I think we should rename the fragment file, even if the file name is not visible to the users. The "query" functions are not equivalent to "polymorphic table functions", which are a much wider abstraction. I'm aware that there is confusion, and people tend to think that PTF is another name for query pass-trough. However, we should actively prevent it, especially that there are more PTFs coming.

Reply:
@jhlodin I am surprised to see this merged in this shape. I strongly recommend rephrasing the fragment so that it speaks specifically about the "query" function, like @martint suggested.

Reply:
@kasiafi Sorry, I missed your comment about renaming the fragment. I do agree that we should do that and will follow up with another PR.

The fragment's contents were discussed and reworked on a call with @electrum . As written it does emphasize that this limitation applies specifically to the query within the function, not the query that invokes the table function itself. We cannot explicitly name the `query` function, unfortunately, because that function name is not consistent across all connectors that support this functionality. So this is the compromise.

Reply:
@kasiafi Created https://github.com/trinodb/trino/pull/14807 as a follow-up